frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -118,7 +118,7 @@ const Group = ({ heading, open = false, updateOpenGroup, closeGroup, children }:
       >
         <Avatar>{heading.charAt(0)}</Avatar>
         <GroupHeading align="center">{heading}</GroupHeading>
-        <Popper open={open} onClickAway={closeGroup} anchorRef={anchorRef}>
+        <Popper open={open} onClickAway={closeGroup} anchorRef={anchorRef} id="workflow-options">
```

Comment:
is this going to be updated too?
```suggestion
        <Popper open={open} onClickAway={closeGroup} anchorRef={anchorRef} id={id}>
```

Reply:
that's passing the ID in to the component that now allows an ID